packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx

```diff
@@ -34,7 +34,9 @@ const PaypalPaymentLegalBlurb = () => (
             href="https://www.paypal.com/webapps/mpp/ua/privacy-full"
             target="_blank"
             rel="noopener noreferrer"
-          ></a>
+          >
+            Paypal privacy policy
```

Comment:
Q: Should these be localized?

Reply:
Also, should be "PayPal", per https://www.paypal.com/

Reply:
This seems to already be localized. This change would just add a default value in case no translation is provided. I don't think this is strictly necessary, but I figured it'd be the most appropriate fix for the linter warning. (They talk a bit more about [React Overlays here](https://github-wiki-see.page/m/projectfluent/fluent.js/wiki/React-Overlays))

Good catch on the PayPal naming. Looks like in the translation we've got it correct, so it's just the default values that needed fixing.